cpp/test/rank.cc

```diff
@@ -51,3 +51,18 @@ TEST(RankTest, TestRankDescription) {
   ASSERT_FALSE(a.isFlush());
   ASSERT_TRUE(b.isFlush());
 }
+
+TEST(RankTestStraights,TestRankDescriptionStraights){
+  Rank a=EvaluateCards("As","Kc","Qd","Jd","Th");
+  Rank b=EvaluateCards("Ks","Qc","Jd","Td","9h");
+  Rank c =EvaluateCards("5h","4d","3d","2c","As");
```

Comment:
The code is correct in the whole block. However, please mind the spaces. Particularly, there should be spaces around the equal sign, and after the commas.